wremnants/muon_calibration.py

```diff
@@ -490,8 +489,8 @@ def transport_smearing_weights_to_reco(
         if 'muonScaleSyst_responseWeights_gensmear' in proc_hist.keys():
             msv_sw_gen_smear = proc_hist['muonScaleSyst_responseWeights_gensmear'].get()
         else:
-            warning.warn(f"Histogram 'muonScaleSyst_responseWeights_gensmear' not found in {proc}")
-            warning.warn("smearing weights not transported to RECO kinematics")
+            warnings.warn(f"Histogram 'muonScaleSyst_responseWeights_gensmear' not found in {proc}")
+            warnings.warn("smearing weights not transported to RECO kinematics")
```

Comment:
Can we remove warnings and add logger.warning?

Reply:
done, thanks for the reminder